networking_generic_switch/devices/netmiko_devices/__init__.py

```diff
@@ -58,23 +59,39 @@ def _format_commands(self, commands, **kwargs):
                                                       args=kwargs)
         return cmd_set
 
-    def send_commands_to_device(self, cmd_set):
+    def send_commands_to_device(self, cmd_set, num_retries=3, wait_on_fail=15):
         if not cmd_set:
             LOG.debug("Nothing to execute")
             return
-
+        for count in range(num_retries):
```

Comment:
Why do we need the multiple retries? I think I missed that bit of the conversation.